markdown/solace-helm-node-samples/Helm-Node-Samples.md

```diff
@@ -0,0 +1,95 @@
+author: Paul Kondrat
+summary: PubSub+ Node.js Samples using Helm / Kubernetes with a Desktop Windows Dev Environment 
+id: helm-node-samples
+tags: 
+categories: Helm
+environments: Web
+status: Published
+feedback link: https://github.com/SolaceDev/solace-dev-codelabs
+analytics account: UA-3921398-10
+
+# PubSub+ Node.js Samples with Helm / Kubernetes
+
+## Overview
+Duration: 0:01:00
+
+In a previous CodeLab (Link to other CodeLab) a desktop Kubernetes development environment was setup using Docker Desktop, Windows Subsystem for Linux, Helm and Visual Studio Code.  That Codlab also used the environment to launch a dev instance of PubSub+ uising the [PubSub+ Helm Charts](https://hub.helm.sh/charts/solace).  This CodeLab will use the dev instance installed in the previous CodeLab to run some Node.js sample applications using Kubernetes and Helm.
```

Comment:
1. (Link to other CodeLab): Adding comment here to remember to update this when the env codelab is up on prod